.github/actions/setup-environment-action/action.yml

```diff
@@ -42,13 +42,13 @@ runs:
       if: ${{ inputs.python-version != '' }}
       uses: actions/setup-python@v4
       with:
-        python-version: ${{ inputs.python-version }}
+        python-version: ${{ inputs.python-version == 'default' && '3.8' || inputs.python-version }}
```

Comment:
Thanks! 
What will be the value if python-version is not specified explicitly by a suite? Should we make it a mandatory param?

Reply:
Alternatively, we could consider making it optional:

python-version: ${{ inputs.python-version || '3.8'  }}

and default to 3.8 if not specified.

Reply:
If python-version not specified, this action won't install python.

I believe our self-hosted runner has pre-installed Java and Python (possibly also go?), so it will default to the runner's java/python/go.

If we do `python-version: ${{ inputs.python-version || '3.8' }}`, this will cause python always get installed when running this action